### PR TITLE
Migrate from Forestry to TinaCMS [RAM-252]

### PR DESCRIPTION
**Pull request by DigitalSuey**

### CODE REVIEW

**Comment by DigitalSuey** [comment]
⚠️ Don't forget to rebase and make sure the new pages work locally before merging

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-redant.com.au
+staging.redant.com.au
```
Comment (jsntv200) [inline CNAME:1]: Remember to set this back to redant.com.au

File: .github/workflows/build-and-deploy.yml [hunk 2]
```diff
@@ -1,59 +1,82 @@
-name: Build and Deploy
+# This workflow uses actions that are not certified by GitHub.
```
Comment (jsntv200) [inline .github/workflows/build-and-deploy.yml:1]: Rebase against master to remove this file which is already in master

**Review by jsntv200 — CHANGES_REQUESTED** [review]
Im inclined to split this into 2 PR's. The first is just the markdown changes, these are dafe to go and probably good to merge. The second PR would be just the TinaCMS stuff if we decide to go with it.

File: .github/workflows/build-and-deploy.yml (hunk 2 above)
Comment (DigitalSuey) [reply .github/workflows/build-and-deploy.yml]: Fixed in [750d657](https://github.com/red-ant/redant-home/pull/307/commits/750d657502d392cf47bf9d186bc35ae0009bc17f)

**Review by jsntv200 — COMMENTED** [review]
All good, just the CNAME to revert.

Don't bother with it for the moment but the config.js got pretty big. Might be worth splitting that out into individual files like the old front_matter templates then load and merge into config.

**Comment by DigitalSuey** [comment]
All markdown changes were moved to https://github.com/red-ant/redant-home/pull/311

File: CNAME (hunk 1 above)
Comment (DigitalSuey) [reply CNAME]: Done in [de86fd4](https://github.com/red-ant/redant-home/pull/307/commits/de86fd4919ea22cbe7ee15c8fc1ff01c852a1d3b)

**Comment by DigitalSuey** [comment]
**Note:** Forestry.io end of life is on 22/03/2023

**Pull request merged**